src/validator.directive.js

```diff
@@ -162,6 +163,8 @@
     return {
       restrict: 'A',
       require: 'ngModel',
+      validFunc: validFunc,
```

Comment:
Actually I will remove this since the exposition still wont override the closure.